onecgiar-pr-client/src/app/pages/ipsr/pages/innovation-package-detail/pages/ipsr-innovation-use-pathway/pages/step-n2/pages/complementary-innovation/components/table-innovation/table-innovation.component.html

```diff
@@ -1,172 +1,219 @@
-                *ngIf="complementaries">
-                <app-pr-input [readOnly]="isReadonly" style="width: 47%;margin-top: -40px;margin-left: 20px;"
-                    label="Other enabler functions" [required]="false"
-                    [(ngModel)]="this.informationComplentary.other_funcions" placeholder=""></app-pr-input>
-            </div>
-
-            <app-pr-field-header label="Reference materials" [required]="false" description='
-            Max 3.'></app-pr-field-header>
-
-
-            <div style="display: flex; flex-direction: row;"
-                *ngFor="let item of this.informationComplentary.referenceMaterials; let i = index">
-
-                <div
-                    style="background-color:aliceblue ; width: 5%; display: flex; align-items: center; justify-content: center; color: black;">
-                    {{i+1}}</div>
-                <div style="width: 95%;"><app-pr-input placeholder="Link" [(ngModel)]="item.link"
-                        [readOnly]="isReadonly"> </app-pr-input></div>
-            </div>
-            <app-pr-button text="Add reference material" padding="medium" *ngIf="isReadonly == false"
-                (click)="addNewInput()" [ngClass]="{'globalDisabled': statusAdd}"></app-pr-button>
-
-
-            <app-pr-radio-button
-                label="Are you aware of any projects or organizations already working on this complementary innovation/ enabler/ solution?"
-                [options]="this.awareOptions" optionLabel="name" optionValue="value" [required]="false"
-                [(ngModel)]="this.informationComplentary.projects_organizations_working_on_innovation"
-                (ngModelChange)="this.informationComplentary.specify_projects_organizations = null"
-                [readOnly]="isReadonly">
-            </app-pr-radio-button>
-            <div style="margin-top: 10px;"></div>
-            <app-pr-textarea *ngIf="this.informationComplentary.projects_organizations_working_on_innovation"
-                [readOnly]="isReadonly" placeholder="Please specify..."
-                description="Please specify those projects or organizations."
-                [(ngModel)]="this.informationComplentary.specify_projects_organizations"></app-pr-textarea>
-
+      <app-pr-input
+        placeholder="Complementary innovation long title"
+        [maxWords]="30"
+        [required]="true"
+        [readOnly]="isReadonly"
+        [(ngModel)]="this.informationComplentary.title">
+      </app-pr-input>
+
+      <app-pr-field-header
+        label="Description"
+        [required]="false"
+        description="
+          <li> Ensure the description is understandable for a non-specialist reader.</li>
+          <li> Avoid acronyms and technical jargon</li>
+          <li> Avoid repetition of the title</li>
+          <li> Try to explain in clear language what kind of challenge this innovation/ enabler/ solution addresses, and what kind of solution it offers</li>"></app-pr-field-header>
```

Comment:
@xKeCo wouldn't it be better to move these narratives with HTML to the component?